src/view/components/rule-engine/rule-condition-group-component.ts

```diff
@@ -17,7 +17,7 @@ import {ServerSideTypeModel} from "../../../api/rule-engine/ServerSideFieldModel
   selector: 'condition-group'
 })
 @View({
-  template: `<div flex layout="column" class="cw-rule-group">
+  template: `<div class="cw-rule-group">
```

Comment:
How does this container get its layout flex-direction value?  If you inspect in chrome is that attribute set?

https://css-tricks.com/snippets/css/a-guide-to-flexbox/